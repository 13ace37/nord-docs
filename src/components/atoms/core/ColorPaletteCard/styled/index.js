/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import Card from "./Card";
import CardWrapper from "./CardWrapper";
import Color from "./Color";
import ColorBox from "./ColorBox";
import FloatingTitle from "./FloatingTitle";
import Label from "./Label";
import Palette from "./Palette";
import Title from "./Title";

export { Card, CardWrapper, Color, ColorBox, FloatingTitle, Label, Palette, Title };
