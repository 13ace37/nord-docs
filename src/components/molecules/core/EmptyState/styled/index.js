/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import Illustration, { illustrationVariants } from "./Illustration";
import Headline from "./Headline";
import Subline from "./Subline";
import Wrapper from "./Wrapper";
import TextWrapper from "./TextWrapper";

export { Headline, Illustration, Subline, TextWrapper, Wrapper, illustrationVariants };
