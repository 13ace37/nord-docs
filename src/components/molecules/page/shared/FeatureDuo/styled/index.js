/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import Actions from "./Actions";
import Headline from "./Headline";
import Subline from "./Subline";
import Text from "./Text";
import Visualization from "./Visualization";
import Wrapper from "./Wrapper";

export { Actions, Headline, Subline, Text, Visualization, Wrapper };
