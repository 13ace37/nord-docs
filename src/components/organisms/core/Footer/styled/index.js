/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import Category from "./Category";
import Copyright from "./Copyright";
import Grid from "./Grid";
import Metadata from "./Metadata";
import OrganizationBrand from "./OrganizationBrand";
import OrganizationBrandCaption from "./OrganizationBrandCaption";
import Sitemap from "./Sitemap";
import SitemapCategory from "./SitemapCategory";
import SitemapLink from "./SitemapLink";
import SitemapList from "./SitemapList";
import SocialNetworking from "./SocialNetworking";

export {
  Category,
  Copyright,
  Grid,
  Metadata,
  OrganizationBrand,
  OrganizationBrandCaption,
  Sitemap,
  SitemapCategory,
  SitemapLink,
  SitemapList,
  SocialNetworking
};
