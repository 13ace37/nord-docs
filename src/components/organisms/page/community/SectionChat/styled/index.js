/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import ChatCard from "./ChatCard/ChatCard";
import Grid from "./Grid";

export { ChatCard, Grid };
