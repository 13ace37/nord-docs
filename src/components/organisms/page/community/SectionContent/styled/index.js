/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import Grid from "./Grid";
import RedditCard from "./RedditCard/RedditCard";
import StackOverflowCard from "./StackOverflowCard/StackOverflowCard";
import WaveDivider from "./WaveDivider";

export { Grid, RedditCard, StackOverflowCard, WaveDivider };
