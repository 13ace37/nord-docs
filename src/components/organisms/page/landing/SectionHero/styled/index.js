/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

/**
 * @file Provides shared components for the landing page orgaism components.
 * @author Arctic Ice Studio <development@arcticicestudio.com>
 * @author Sven Greb <development@svengreb.de>
 * @since 0.6.0
 */

import Cloud1 from "./Cloud1";
import Cloud2 from "./Cloud2";
import Cloud3 from "./Cloud3";
import DecorationCanvas from "./DecorationCanvas";
import WaveDivider from "./WaveDivider";

export { Cloud1, Cloud2, Cloud3, DecorationCanvas, WaveDivider };
