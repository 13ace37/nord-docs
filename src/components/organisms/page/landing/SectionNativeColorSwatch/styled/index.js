/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import DecorationCanvas from "./DecorationCanvas";
import SketchTools from "./SketchTools";
import VectorDesignCollage from "./VectorDesignCollage";
import WaveDivider from "./WaveDivider";

export { DecorationCanvas, SketchTools, VectorDesignCollage, WaveDivider };
