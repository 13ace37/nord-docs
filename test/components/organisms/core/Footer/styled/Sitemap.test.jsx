/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import React from "react";
import { stripUnit } from "polished";

import { renderWithTheme } from "nord-docs-test-utils";
import { Sitemap } from "organisms/core/Footer/styled";

describe("theme styles", () => {
  test("has top and bottom margin", () => {
    const { container } = renderWithTheme(<Sitemap />);
    const margingTop = stripUnit(getComputedStyle(container.firstChild).marginTop);
    const marginBottom = stripUnit(getComputedStyle(container.firstChild).marginBottom);

    expect(margingTop).toBeGreaterThan(0);
    expect(marginBottom).toBeGreaterThan(0);
  });

  test("has flexbox column layout styles", () => {
    const { container } = renderWithTheme(<Sitemap />);

    expect(container.firstChild).toHaveStyleRule("display", "flex");
    expect(container.firstChild).toHaveStyleRule("flex-direction", "column");
  });

  test("matches the snapshot", () => {
    const { container } = renderWithTheme(<Sitemap />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
