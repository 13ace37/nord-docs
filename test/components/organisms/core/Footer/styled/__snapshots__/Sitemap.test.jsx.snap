// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`theme styles matches the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-top: 1em;
  margin-bottom: 1em;
}

@media (min-width:37.5em) {
  .c0 {
    -webkit-align-items: unset;
    -webkit-box-align: unset;
    -ms-flex-align: unset;
    align-items: unset;
    margin-top: 0;
    margin-bottom: 0;
  }
}

<div
  class="c0"
/>
`;
